src/components/actions/NetlifyFormCallToAction.js

```diff
@@ -37,7 +51,7 @@ const NetlifyFormCallToAction = ({
   placeholderText = 'Work email',
   buttonText = 'Click here',
   subFormMessage = 'We will never sell or share your email address.',
-  netlifyFormName = FORM_NAMES.notifyMe,
+  netlifyFormName = FORM_NAMES.getInstance,
```

Comment:
The `notifyMe` form is legacy so I am updating this default to a form we more recently used.